desktop/libs/indexer/src/indexer/indexers/sql_tests.py

```diff
@@ -834,3 +834,36 @@ def test_create_table_from_local():
   `dist` bigint);'''
 
   assert_equal(statement, sql)
+
+
+def test_create_table_from_local_mysql():
+  source = {
+    'path': '/apps/beeswax/data/tables/us_population.csv',
```

Comment:
Nice!

Main issue though is that it will tie tests to demo data, and we can't usually test corner cases (i.e. adding a boolean column...) and reusing the same for the other dialects.

For sure we will need to mock the File data with Magic Mock at some point, good for now but to refactor when doing test for hive, impala, phoenix dialects